libs/unity/library/Runtime/Scripts/NativeRender/NativeRenderingPluginUpdate.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.WebRTC.UnityPlugin
+{
+	public class NativeRenderingPluginUpdate : MonoBehaviour
```

Comment:
This should be internal, since it should only be visible to `NativeVideoRenderer`. Also it needs some comments on what it does.

Reply:
A monobehaviour can't be nested inside of another class, but I changed it to internal.